bin/mathjaxify

```diff
@@ -94,9 +103,27 @@ AssistiveMmlHandler(RegisterHTMLHandler(adaptor));
 //
 //  Create input and output jax and a document using them on the content from the HTML file
 //
-const tex = new TeX({packages: AllPackages});
-const svg = new SVG({fontCache: 'global', exFactor: argv.ex / argv.em});
-const html = mathjax.document(htmlfile, {InputJax: tex, OutputJax: svg});
+function getInput() {
+    const options = {packages: AllPackages};
+    if (argv.singleDollars) {
+        options.inlineMath = [['$', '$'], ['\\(', '\\)']]
+    }
+    return new TeX(options);
+}
+
+function getOutput() {
+    const options = {exFactor: argv.ex / argv.em};
+    switch (argv.output) {
+        case 'SVG':
+            return new SVG({...options, fontCache: 'global'});
+        case 'CommonHTML':
+            return new CHTML({...options, fontURL: 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2'});
```

Comment:
The URL seems dead, at least when opening it in a browser. This works there, but not sure about which one is correct for resource loading:
```suggestion
            return new CHTML({...options, fontURL: 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/'});
```

Reply:
I did test that the fonts load properly with that URL. I think `fontURL` is meant to be a URL prefix rather than a full URL. For instance, see the `fontURL` in the MathJax docs: http://docs.mathjax.org/en/latest/options/output/chtml.html